### PR TITLE
Update break-and-continue-statements.md

### PR DESCRIPTION
**Pull request by nem035**
Polish the insight

### CODE REVIEW

**Comment by enki-hq** [comment]
Failed parsing files: 
  - python/python-core/looping/break-and-continue-statements.md: 
 > 	No valid answer list in question node

**Comment by enki-hq** [comment]
Failed parsing files: 
  - python/python-core/looping/break-and-continue-statements.md: 
 > 	No valid answer list in question node

**Comment by enki-hq** [comment]
Failed parsing files: 
  - python/python-core/looping/break-and-continue-statements.md: 
 > 	No valid answer list in question node

File: python/python-core/looping/break-and-continue-statements.md [hunk 1]
```diff
@@ -82,11 +91,18 @@ Which keyword is used to skip the current code block, and to resume execution at
...
 
 ???
 
 - `break`
 - `continue`
 - `next`
  
+---
+
+### Footnotes
```
Comment (nem035) [inline python/python-core/looping/break-and-continue-statements.md:104]:
```suggestion
## Footnotes
```

**Comment by enki-hq** [comment]
### Changelog

| File Name | Error Message |
|:-:|:-:|
| Pull Request | Couldn't find a CHANGELOG entry for this pull request. |

> If you notice anything wrong with these rules contact andrei@enki.com.

**Comment by enki-hq** [comment]
### Insight previews:
- [ ] [enkidevs/curriculum/python/python-core/looping/break-and-continue-statements.md](https://app.enkipro.com/public/preview-insight/enkidevs/curriculum/python/python-core/looping/break-and-continue-statements.md?ref=605c458c5b981389304eaf2392cc4b3cdc545834)

**Review by Stefan-Stojanovic — APPROVED** [review]

**Pull request merged**